src/main/java/com/googleinterns/zoomtube/transcriptParser/TranscriptParser.java

```diff
@@ -164,4 +164,17 @@ private Entity createTranscriptLineFromElement(Key lectureKey, Element transcrip
     return TranscriptLineUtil.createEntity(
         lectureKey, lineContent, lineStartMs, lineDurationMs, lineEndMs);
   }
+
+  /**
+   * Cleans the {@code content} of a transcript line by unescaping XML characters, and
+   * removing newlines.
+   */
+  private String cleanupTranscriptContent(String content) {
+    String unescapedXml = StringEscapeUtils.unescapeXml(content);
+
+    // We ignore '\r' for now.
+    String cleanedContent = unescapedXml.replace('\n', ' ');
```

Comment:
Is there a reason we replace with a space and not an ''?

Reply:
Yeah.  Example transcript line: `This is a super long transcript line\nthat has a newline.`